Automattic/sensei
# Update question post type to not be publicly queryable

Fixes #5215

### Changes proposed in this Pull Request

* It makes the question not publicly queryable - Kudos @gkaragia for the solution in the issue comments!

### Testing instructions


* Create a course with a lesson containing a quiz.
* Add at least a question.
* Try to access `https://{your-domain}/question/{question-slug}` and make sure it doesn't load.
* As a student, answer the quiz and make sure it works properly.
* As a teacher grade the quiz and make sure it works properly.

### Removed hooks

- `sensei_question_slug` - as we removed the `rewrite`, the filter for the question slug was also removed since it's not used.
